tests/references/empty_xnat.ttl

```diff
@@ -0,0 +1,8 @@
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<https://xnat.bmia.nl> a dcat:catalog ;
```

Comment:
Types in RDF are capitalized by convention: in this case it should be `dcat:Catalog`.

Reply:
Thanks for catching this, it was also a bug in the xnat_to_Catalog function.